internal/cmd/app.go

```diff
@@ -89,7 +89,9 @@ func NewApp(o *globals.GlobalOpts) *cli.App {
 		if err := setEnvoyVersionsURL(o, envoyVersionsURL); err != nil {
 			return err
 		}
-		o.FuncEVersions = envoy.NewFuncEVersions(o.EnvoyVersionsURL, o.Platform, o.Version)
+		if o.FuncEVersions == nil {
```

Comment:
I think we'd better put comment about this brunch! - this only happens in tests, right?

Reply:
sorry typo:  *branch 😄

Reply:
I got it @mathetake! I'm hungry already!